src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -471,6 +480,11 @@
 helps['deployment sub wait'] = """
 type: command
 short-summary: Place the CLI in a waiting state until a deployment condition is met.
+examples:
+  - name: Place the CLI in a waiting state until a deployment condition is met. (autogenerated)
+    text: |
+        az deployment sub wait --created --name MyDeployment --updated
```

Comment:
`--created` and `--updated` are not recommended to be used at the same time, because they are both judgments of state in different scenarios.

Reply:
What's the correct example?

Reply:
Is it a valid command to use `--created` and `--updated` at the same time?

I will keep `--created` if it's not recommended used them at the same time.

Reply:
Maybe we should modify this part to restrict user not wait more than one status changed.